app/src/main/java/org/openobservatory/ooniprobe/common/ResubmitTask.java

```diff
@@ -152,6 +153,8 @@ protected Boolean doInBackground(Integer... params) {
                 m.result.load();
                 if(!d.measurementsManager.reSubmit(m, session)){
                     errors++;
+                }else {
+                    m.deleteEntryFile(getActivity());
```

Comment:
By reading the code inside `Measurement.java`, I see that we have the `hasReportFile` method and then we have `getEntryFile` and `deleteEntryFile`. Could we refactor the last two methods to become `getReportFile` and `deleteReportFile`? I think the "report" based naming is more clear than the "entry" based naming, judging on the terminology I am aware of and I reckon we have been using.